src/reporting/clients/IExtractionClient.ts

```diff
@@ -0,0 +1,58 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+import { AccessToken } from "@itwin/core-bentley";
+import { ExtractionLog, ExtractionRun, ExtractionStatus } from "../interfaces/ExtractionProcess";
+import { EntityListIterator } from "../iterators/EntityListIterator";
+
+export interface IExtractionClient {
+  /**
+   * Gets Logs of an Extraction Run.
+   * @param {string} jobId Unique Identifier of the Extraction Run.
+   * @param {string} accessToken OAuth access token with scope `insights:read`
+   * @param {number} top the number of entities to pre-load.
+   * @memberof ReportingClient
+   * @link https://developer.bentley.com/apis/insights/operations/get-extraction-logs/
+   */
+  getExtractionLogs(
+    accessToken: AccessToken,
+    jobId: string,
+    top?: number
+  ): Promise<ExtractionLog[]>,
+  /**
```

Comment:
Add empty line between method and the next method's documentation comments for better readability.

Reply:
added empty lines